src/MIDebugEngine/Engine.Impl/Breakpoints.cs

```diff
@@ -192,9 +192,16 @@ private static BindResult EvalBindResult(Results bindResult, AD7PendingBreakpoin
             Debug.Assert(bkpt.FindString("type") == "breakpoint");
 
             string number = bkpt.FindString("number");
+            string warning = bkpt.TryFindString("warning");
             string addr = bkpt.TryFindString("addr");
 
-            PendingBreakpoint bp = new PendingBreakpoint(pbreak, number, StringToBreakpointState(addr));
+            PendingBreakpoint bp;
+            if (!string.IsNullOrEmpty(warning))
```

Comment:
Is `warning` only sent in the case of a mismatch of checksums?

Could it be the case that someone adds a warning but doesn't want the breakpoint state to go to `MIBreakpointState.Pending`?